### PR TITLE
add caption for kayakkat

### PR DESCRIPTION
**Pull request by kayakkat**
CHANGE2 changed kayakkat to include Whoa! in caption

## Tasks remaining
- [ ] first task

### CODE REVIEW

File: _posts/0000-01-05-kayakkat.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ title: "Welcome to our slide deck!"
 ---
 
 Caption here
-
+Whoa!
```
Comment (js3263827) [inline _posts/0000-01-05-kayakkat.md:6]: 👍

Comment (githubteacher) [inline _posts/0000-01-05-kayakkat.md:6]: Far out! 👋 🌊 😮

File: _posts/0000-01-05-kayakkat.md [hunk 2]
```diff
@@ -4,6 +4,6 @@ title: "Welcome to our slide deck!"
 ---
 
 Caption here
```
Comment (githubteacher) [inline _posts/0000-01-05-kayakkat.md]: Hey! I think we can remove the "Caption here" line. What do you think? 😄 ✨

**Review by js3263827 — CHANGES_REQUESTED** [review]
@kayakkat merge when you want.

**Review by githubteacher — CHANGES_REQUESTED** [review]
@kayakkat this looks great! 

I left a suggested action for you, but you can feel free to merge without fixing it if you want!

**Review by js3263827 — APPROVED** [review]

**Review by githubteacher — APPROVED** [review]

**Pull request merged**